internal/modelconfig/types/configuration.go

```diff
@@ -8,6 +8,8 @@ type ClientSideProviderConfig struct {
 	// We currently do not have any known client-side provider configuration.
 	// But later, if anything needs to be provided to Cody clients at the
 	// provider-level it will go here.
+	// Client-side provider configuration is sent to clients, and so should
```

Comment:
```suggestion
	//
	// Client-side provider configuration is sent to clients, and so should
```

Yes this is totally unimportant and definitely a matter of personal preference. Feel free to ignore. But hopefully just muttering "What's wrong with Chris?!? ugh." and clicking "commit changes" isn't too much of an inconvenience 😅